src/NzbDrone.Api/Calendar/CalendarFeedModule.cs

```diff
@@ -92,6 +93,10 @@ private Response GetCalendarFeed()
                 ProductId = "-//sonarr.tv//Sonarr//EN"
             };
 
+            var calendarName = "Sonarr TV Schedule Calendar";
+            calendar.AddProperty(new CalendarProperty("NAME", calendarName));
```

Comment:
Is this used by a specific client? Looks like `X-WR-CALNAME` is fairly well used, but didn't find anything on name.

Reply:
It's a good question. I wasn't too sure either. I looked at a couple of iCal examples and a few had both.

Let me see if I can provide some samples and references.

Reply:
So X-WR-CALNAME does appear to be used widely and is classed as an extension, but the name property does appear to exist in some form.

https://tools.ietf.org/html/rfc7986#section-5.1

I can drop it, if you feel its not worth including, but both appear to be used in some cases, here's an example:

* https://www.formula1.com/sp/static/f1/2017/calendar/ical.ics

Reply:
Thanks, leaving it is fine with me.